examples/store/db/seeds.ts

```diff
@@ -0,0 +1,30 @@
+import db from "./index"
```

Comment:
it would be nice to reduce a touch of boilerplate from the `seeds.ts` file...is there a typesafe way that we could pass `db` into the exported function as an argument rather than importing directly? this way feels a bit like you need to know about how our DB handling works.

Reply:
Hmm I was thinking about it, but this seamed easier to type. I think it would be a lot harder if we required people to add types to the function they export.

```ts
import {DBType} from 'somewhere' 

const seed = async (db: DBType) => {
	// ...
}

export default seed;
```

We could maybe have a wrapper that added types?

```ts
import {createSeeds} from 'blitz' 

const seed = createSeeds(async (db) => {
	// ...
})

export default seed;
```

Or just keep the current setup

```ts
import {db} from './index'

const seed = async () => {
	// ...
}

export default seed;
```

@flybayer @aem Thoughts?

Reply:
I think importing the db (like you have now) is the perfect way to do it. Not magical. Very clear what's happening.

Reply:
Ok, closing this for now then